_field-types/dynamic.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: default
+title: Dynamic parameter
+nav_order: 10
+redirect_from:
+  - /opensearch/dynamic/
+---
+
+# Dynamic parameter
+
+The `dynamic` parameter specifies whether new detected fields can be added dynamically to the mapping. It accepts the parameters listed in the following table.
+
+Parameter | Description 
+:--- | :--- 
+`true`  | Specfies that new fields can be added dynamically to the mapping. Default is `true`.
+`false` | Specifies that new fields cannot be added dynamically to the mapping. If a new field is detected, then it is not indexed or searchable but can be retrieved from the `_source` field.
+`strict` | Throws an exception. The indexing operation fails when new fields are detected.
+`strict_allow_templates` | Adds new fields matching dynamic templates if the new fields match predefined dynamic templates in the mapping; otherwise, indexing fails.
+
+--- 
+
+## Example: Create index with dynamic parameter set to `true`
+
+1. Create an index with `dynamic` set as `true` using the following request:
+
+```json
+PUT testindex1
+{
+  "mappings": {
+    "dynamic": true
+  }
+}
+```
+{% include copy-curl.html %}
+
+2. Index a document with an object field `patient` containing two string fields using the following request:
+
+```json
+PUT testindex1/_doc/1
+{ 
+  "patient": { 
+    "name" : "John Doe",
+    "id" : "123456"
+  } 
+}
+```
+{% include copy-curl.html %}
+
+3. Check the mapping using the following request:
```

Comment:
Do we mean something slightly more precise than "Check"?

Reply:
Verify